src/runtime/HalideBuffer.h

```diff
@@ -2061,9 +2061,8 @@ class Buffer {
     }
 
     template<typename... Args>
-    HALIDE_ALWAYS_INLINE
-        storage_T *
-        address_of(Args... args) const {
+    HALIDE_ALWAYS_INLINE storage_T *
```

Comment:
imo if this is now two lines, it would be better with the return type on the second line instead of the first.

Reply:
It wouldn't let me break there, but it seems happy with everything on one line, which is fine in this case. If I put a `//` break after `HALIDE_ALWAYS_INLINE`, it still insists on breaking after `storage_T *`.